admin/spec/components/previews/solidus_admin/ui/tab/component_preview.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# @component "ui/tab"
+class SolidusAdmin::UI::Tab::ComponentPreview < ViewComponent::Preview
+  include SolidusAdmin::Preview
+
+  def overview(text: "text")
+    render_with_template locals: { text: text }
+  end
+
+  # @param text text
+  # @param size select { choices: [s, m, l] }
+  # @param state select { choices: [default, active, disabled] }
```

Comment:
What about using a `toggle` type? Given that `default` & `active` mean the same, and we indeed act as if the param was a boolean (we don't consider the value can be `:default`). We already used this type in the item component preview: https://github.com/solidusio/solidus/blob/a81065132489f025ff9c42e4228e3aecaffc766e/admin/spec/components/previews/solidus_admin/sidebar/item/component_preview.rb#L13